### PR TITLE
GML-1117: SSL cert fixes

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

File: pyTigerGraph/pyTigerGraphAuth.py [hunk 1]
```diff
@@ -226,33 +226,25 @@ def getToken(self, secret: str, setToken: bool = True, lifetime: int = None) ->
...
-                        "/requesttoken?secret=" + secret +
-                        ("&lifetime=" + str(lifetime) if lifetime else ""), verify=False).text)
+                res = json.loads(requests.request("GET", self.restppUrl +
+                    "/requesttoken?secret=" + secret +
+                    ("&lifetime=" + str(lifetime) if lifetime else ""), verify=False).text)
                 if not res["error"]:
                     success = True
-            except:
-                success = False
+            except Exception as e:
```
Comment (billshitg) [inline pyTigerGraph/pyTigerGraphAuth.py:234]: just nitpicking: if catch and raise without doing something else, why not remove try...except altogether?

File: pyTigerGraph/pyTigerGraphAuth.py [hunk 2]
```diff
@@ -226,33 +226,25 @@ def getToken(self, secret: str, setToken: bool = True, lifetime: int = None) ->
...
-                        data=json.dumps(data)).text)
-                else:
-                    res = json.loads(requests.post(self.restppUrl + "/requesttoken",
-                        data=json.dumps(data), verify=False).text)
-            except:
-                success = False
+
+                res = json.loads(requests.post(self.restppUrl + "/requesttoken",
+                    data=json.dumps(data), verify=False).text)
+            except Exception as e:
```
Comment (billshitg) [inline pyTigerGraph/pyTigerGraphAuth.py:246]: same as above

**Review by billshitg — APPROVED** [review]

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SKIPPED, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/483/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

**Pull request merged**